src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Display />
+        <ButtonPanel />
+      </div>
```

Comment:
- [ ] Also please wrap Display and ButtonPanel components in a `React.Fragment` shorter syntax.